pages/content.py

```diff
@@ -19,3 +20,11 @@ def wait_for_page_to_load(self):
     @property
     def title(self):
         return self.find_element(*self._title_locator).text
+
+    @property
+    def not_converted_yet(self):
+        return self.find_element(*self._ncy_locator).text
```

Comment:
Your returning the text here. The return value of this is the text inside the locator. It is literally returning a string `NOT_CONVERTED_YET: caption`.

Your intuition on this is really good by the way but you don't want the text. You to return `is_element_displayed`.